src/pages/journal/journal.reducer.ts

```diff
@@ -23,6 +23,9 @@ export function journalReducer(state = initialState, action: journalActions.Jour
         error: { message: '', status: 0, statusText: '' },
       };
     case journalActions.CANDIDATE_DETAILS_SEEN:
+      if (state.checkComplete.findIndex(checkComplete => checkComplete.slotId === action.slotId) !== -1) {
```

Comment:
Just want to mention that I think there is too much logic in the reducer with this. No change required, only if you prefer to do